source/includes/mongodb-compatibility-table-pymongo.rst

```diff
@@ -21,7 +21,7 @@
      - ✓
      - ✓
      - ✓
-     - ✓
+     - 
```

Comment:
Are there language compatibility changes needed too?

Reply:
No, the language compat changes were already in master for this version